aopy/precondition/base.py

```diff
@@ -598,6 +598,62 @@ def bin_spikes(data, fs, bin_width):
     binned_spikes = binned_spikes/bin_width # convert from [spikes/bin] to [spikes/s]    
     return binned_spikes
 
+def calc_ks_waveforms(raw_data, sample_rate, spike_times_unit, templates, channel_pos, waveforms_nch=10, time_before=1000., time_after=1000.):
+    '''
+    Calculate waveforms, waveform channels, and positions of units, using templates from kilosort
+    
```

Comment:
Can you add a comment here that this function does not account for drift correction?